wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -0,0 +1,12 @@
+# -*- encoding : utf-8 -*-
+
+require 'decko/engine'
+
+Rails.application.routes.draw do
+
+  if !Rails.env.production? && Object.const_defined?( :JasmineRails )
+    mount Object.const_get(:JasmineRails).const_get(:Engine) => "<%= mount_point %>/specs"
+  end
+
+  mount Decko::Engine => '<%= mount_point %>/'
```

Comment:
wondering about the word "mount_point".  not sure that's how folks will think of it.  it's really just the "deck_path", right?